app/views/individual_principals/_memberships.html.erb

```diff
@@ -31,14 +31,14 @@ See docs/COPYRIGHT.rdoc for more details.
 
 <div class="grid-block">
   <div class="grid-content">
-  <% if @user.memberships.any? %>
+  <% if @individual_principal.memberships.any? %>
     <div class="generic-table--container">
       <div class="generic-table--results-container">
         <table class="generic-table memberships">
           <colgroup>
             <col highlight-col>
             <col highlight-col>
-            <%= call_hook(:view_users_memberships_table_colgroup, user: @user )%>
+            <%= call_hook(:view_placeholder_users_memberships_table_colgroup, placeholder_user: @individual_principal)%>
```

Comment:
And what if we make it even more abstract, such as? Then there would be different hooks called for users and placeholder users. I think that would allow plugins to be more specific.

```
<%= call_hook("view_#{@individual_principal.class.name.underscore.pluralize}_memberships_table_colgroup".to_sym,
                          @individual_principal.class.name.underscore.to_sym => @individual_principal) %>
```

Reply:
extracted to a helper